docs/contributing-guide.md

```diff
@@ -188,3 +188,33 @@ modified the name of a sidebar or the title of a documentation page, type
 <kbd>CTRL</kbd>+<kbd>C</kbd> on the `yarn start` process and run it again.
 Docusaurus doesn't incrementally rebuild changes to the Markdown page
 indices.
+
+## Releasing
+
+- Draft the release notes:
+
+  - You might use `./bin/release_notes.sc` script to generate merged PRs list
+    between two last release tags. It can be run using ammonite:
+
+  ```
+  cs install ammonite
```

Comment:
This is what I was referring to before. If this is truly tied to 2.12, we should switch this.

Reply:
It's not, will remove the requirement!